### DISCUSSION TITLE
how to remove the article summary？

### DISCUSSION THREAD

**Discussion by Floweryu** [vuepress-theme-hope/vuepress-theme-hope#3635] [Q&A] [answered]
![image](https://github.com/vuepress-theme-hope/vuepress-theme-hope/assets/45811382/ab3b678e-b585-432e-96b4-80c3cfc86af0)
`  plugins: {
    blog: {
      excerpt: false,
      excerptLength: 0,
    },
}`
The above code configuration does not take effect

**Comment by Mister-Hope** [comment, marked as answer]
The article excerpt is actually disabled in this case.

The content in pic is actually article description, which is generated by SEO plugin.

**Reply by Mister-Hope** [reply]
This is considered as a bug and will be fixed in new version, only user-defined description is considered as fallback of excerpt now.

**Reply by Floweryu** [reply]
So how to close this article description，in new version？

**Reply by Mister-Hope** [reply]
> This is considered as a bug and will be fixed in new version, only user-defined description is considered as fallback of excerpt now.

yes not released yet

**Reply by raysun0501** [reply]
Hi, Mr. Hope. If you have solved this question, please remind me. Thank you!

**Reply by Mister-Hope** [reply]
I believe this is solved a long time ago.

**Reply by raysun0501** [reply]
okay, thanks Mr. Hope.
I have solved this question by upgrade to the latest version of vuepress-theme-hope.
